.github/actions/download-artifact-retry/action.yaml

```
@@ -0,0 +1,31 @@
+name: Download artifact with retry
+description: Retry wrapper of download-artifact
+inputs:
+  name:
+    description: Artifact name
+    required: false
+  path:
+    description: Artifact path
+    required: false
```

Optional suggestion. We could borrow these from the [action itself](https://github.com/actions/download-artifact/blob/main/action.yml), for niceness.
```suggestion
  name:
    description: 'Name of the artifact to download. If unspecified, all artifacts for the run are downloaded.'
    required: false
  path:
    description: 'Destination path. Supports basic tilde expansion. Defaults to $GITHUB_WORKSPACE'
    required: false
```